packages/fuel-gauge/src/await-execution.test.ts

```diff
@@ -1,90 +1,90 @@
-      args: ['--poa-instant', 'false', '--poa-interval-period', '400ms'],
+    using launched = await launchTestNode({
+      nodeOptions: {
+        args: ['--poa-instant', 'false', '--poa-interval-period', '400ms'],
+      },
     });
-    const nodeProvider = await Provider.create(`http://${ip}:${port}/v1/graphql`);
-    const baseAssetId = nodeProvider.getBaseAssetId();
 
-    const genesisWallet = new WalletUnlocked(
-      process.env.GENESIS_SECRET || randomBytes(32),
-      nodeProvider
-    );
+    const {
+      provider,
+      wallets: [genesisWallet],
+    } = launched;
 
-    const destination = Wallet.generate({ provider: nodeProvider });
+    const destination = Wallet.generate({ provider });
 
-    const transfer = await genesisWallet.createTransfer(destination.address, 100, baseAssetId, {
-      gasLimit: 10_000,
-    });
+    const transfer = await genesisWallet.createTransfer(
+      destination.address,
+      100,
+      provider.getBaseAssetId(),
+      {
+        gasLimit: 10_000,
+      }
+    );
 
     transfer.updateWitnessByOwner(
       genesisWallet.address,
       await genesisWallet.signTransaction(transfer)
     );
 
-    const response = await nodeProvider.sendTransaction(transfer, { awaitExecution: true });
+    const response = await provider.sendTransaction(transfer, { awaitExecution: true });
 
     expect(response.gqlTransaction?.status?.type).toBe('SuccessStatus');
-
-    cleanup();
   });
 
   test.skip('transferring funds with awaitExecution works', async () => {
-    const provider = await Provider.create(FUEL_NETWORK_URL);
-    const baseAssetId = provider.getBaseAssetId();
+    using launched = await launchTestNode({
+      walletsConfig: {
+        amountPerCoin: 1_000_000,
+      },
+    });
 
-    const genesisWallet = new WalletUnlocked(
-      process.env.GENESIS_SECRET || randomBytes(32),
-      provider
-    );
+    const { provider } = launched;
+
+    const {
+      wallets: [genesisWallet],
+    } = launched;
 
     const sendTransactionSpy = vi.spyOn(provider, 'sendTransaction');
 
     const destination = Wallet.generate({ provider });
 
-    await genesisWallet.transfer(
-      destination.address,
-      100,
-      baseAssetId,
-      {
-        gasLimit: 10_000,
-      }
-      // { awaitExecution: true }
-    );
+    await genesisWallet.transfer(destination.address, 100, provider.getBaseAssetId(), {
+      gasLimit: 10_000,
+    });
 
     expect(sendTransactionSpy).toHaveBeenCalledTimes(1);
     const awaitExecutionArg = sendTransactionSpy.mock.calls[0][1];
     expect(awaitExecutionArg).toMatchObject({ awaitExecution: true });
   });
 
-  test.skip('withdrawToBaseLayer works with awaitExecution', async () => {
-    const provider = await Provider.create(FUEL_NETWORK_URL);
-    const genesisWallet = new WalletUnlocked(
-      process.env.GENESIS_SECRET || randomBytes(32),
-      provider
-    );
+  test('withdrawToBaseLayer works with awaitExecution', async () => {
```

Comment:
No but the `sendTransaction` method does, which is ultimately called by `withdrawToBaseLayer` that being said I can't comment on the relevancy of this test.

Reply:
Both of these tests were skipped [here](https://github.com/FuelLabs/fuels-ts/pull/1615#discussion_r1450417265). You can delete them because the discussion #1643, if anything ever happens to it, would reintroduce them.

Reply:
Not sure I get the rational for deleting - resolving this comment, we can reassess later IMO